module/go.mod

```diff
@@ -3,26 +3,27 @@ module github.com/Gravity-Bridge/Gravity-Bridge/module
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.9
-	github.com/cosmos/ibc-go/v3 v3.2.1
-	github.com/ethereum/go-ethereum v1.10.16
-	github.com/evmos/ethermint v0.17.0
+	github.com/cosmos/cosmos-sdk v0.45.14
+	github.com/cosmos/ibc-go/v3 v3.4.0
+	github.com/ethereum/go-ethereum v1.10.19
+	github.com/evmos/ethermint v0.21.0
```

Comment:
hey we have a replace statement for ethermint v0.19 further down, what's the impact there?

Reply:
This mistake has been resolved